### DISCUSSION TITLE
Landmarks Application: player data

### DISCUSSION THREAD

**Discussion by SR-gituser** [mjstarrett/Landmarks#26] [Q&A] [answered]
Hello :) 
A navigation task was designed in Landmarks, and the player data was obtained from 'editor-data' (within the Landmarks folder). 
The game was then exported and packaged from unity to an application (File>Build Settings>Build).
After running the navigation task in the application, where is the player data saved? I've checked all folders including landmarks_Data, but can't seem to find the corresponding files. Any feedback welcome.
Kind regards, SR-gituser.

**Comment by mjstarrett** [comment, marked as answer]
@SR-gituser, Landmarks uses Unity's `Application.persistentDataPath` (see https://docs.unity3d.com/2019.4/Documentation/ScriptReference/Application-persistentDataPath.html). If you haven't changed the Company Name or Product name in the Unity "Project Settings" or any info in the Config that is generated at runtime, you should be able to find it at either at `C:\Users\[Your-Username]\App Data\Local-Low\mjstarrett\Landmarks\default\default\` (Windows) or `/Users/[Your-Username]/Library/Application Support/mjstarrett/Landmarks/default/default/` (MacOS). On MacOS it might also get placed under a directory called `com.mjstarrett.Landmarks` or something similar. Note that the two 'default' directories on each path are defined by the Config (which is generated at runtime unless pre-configured in your scene) properties experiment and subject id; they are set to the string 'default' by... well... by default.

**Reply by SR-gituser** [reply]
Thank you, the player data was located 👍
